app/modules/ads/module.js

```diff
@@ -130,16 +132,21 @@ class Ads {
 
         billTheLizard.projectsHandler.enable('cheshirecat');
         billTheLizard.executor.register('catlapseIncontentBoxad', () => {
-          const slots = Object.keys(context.get('slots'))
-            .filter(slotName => slotName.indexOf('incontent_boxad_') === 0);
+          const slot = `incontent_boxad_${Object.keys(this.cheshirecatPredictions).length + 2}`;
```

Comment:
Why +2?

Reply:
+1: Next incontent_boxad
+2: IncontentBoxads are counted from 1, not 0